spec/core/plugin_spec.rb

```diff
@@ -29,6 +29,22 @@ class LogStash::Filters::LadyGaga < LogStash::Filters::Base
     expect(LogStash::Plugin.lookup("filter", "lady_gaga")).to eq(LogStash::Filters::LadyGaga)
   end
 
+  describe "#inspect" do
+    class LogStash::Filters::MyTestFilter < LogStash::Filters::Base
```

Comment:
Is a personal taste of mine, I like to have this classes either wrap in lets or defined in a helper file somewhere else, just to organize stuff. Defining classes like this in test cases tend to pollute the code, at less in my experience.